'make' fails

make fails with the tests.

```
WDG:ircd.js-master Wes$ make
./node_modules/.bin/mocha --reporter list -c --ui exports test/*.test.js

  1) Channels test bad join (#22)
  2) Channels "after each" hook
  3) Clients test valid WHOIS
  4) Clients "after each" hook
  ․ Protocol test nickname validation: 0ms
  ․ Protocol test channelname validation: 0ms
  ․ Protocol test channelkey validation: 0ms
  ․ Server test connection passwords: 160ms
  ․ Sockets test destroy a socket: 0ms
  ․ Sockets test send garbage: 1ms
  5) Sockets "after each" hook

  6 passing (8s)
  5 failing

  1) Channels test bad join (#22):
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/Users/Wes/Downloads/ircd.js-master/node_modules/mocha/lib/runnable.js:157:19)
      at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)

  2) Channels "after each" hook:
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/Users/Wes/Downloads/ircd.js-master/node_modules/mocha/lib/runnable.js:157:19)
      at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)

  3) Clients test valid WHOIS:
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/Users/Wes/Downloads/ircd.js-master/node_modules/mocha/lib/runnable.js:157:19)
      at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)

  4) Clients "after each" hook:
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/Users/Wes/Downloads/ircd.js-master/node_modules/mocha/lib/runnable.js:157:19)
      at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)

  5) Sockets "after each" hook:
     Uncaught Error: read ECONNRESET
      at errnoException (net.js:901:11)
      at TCP.onread (net.js:556:19)



make: *** [test] Error 5
```

```
WDG:ircd.js-master Wes$ sudo make
./node_modules/.bin/mocha --reporter list -c --ui exports test/*.test.js

  1) Channels test bad join (#22)
  2) Channels "after each" hook
  3) Clients test valid WHOIS
  4) Clients "after each" hook
  ․ Protocol test nickname validation: 1ms
  ․ Protocol test channelname validation: 0ms
  ․ Protocol test channelkey validation: 1ms
  ․ Server test connection passwords: 166ms
  ․ Sockets test destroy a socket: 0ms
  ․ Sockets test send garbage: 1ms
  5) Sockets "after each" hook

  6 passing (8s)
  5 failing

  1) Channels test bad join (#22):
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/Users/Wes/Downloads/ircd.js-master/node_modules/mocha/lib/runnable.js:157:19)
      at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)

  2) Channels "after each" hook:
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/Users/Wes/Downloads/ircd.js-master/node_modules/mocha/lib/runnable.js:157:19)
      at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)

  3) Clients test valid WHOIS:
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/Users/Wes/Downloads/ircd.js-master/node_modules/mocha/lib/runnable.js:157:19)
      at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)

  4) Clients "after each" hook:
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/Users/Wes/Downloads/ircd.js-master/node_modules/mocha/lib/runnable.js:157:19)
      at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)

  5) Sockets "after each" hook:
     Uncaught Error: read ECONNRESET
      at errnoException (net.js:901:11)
      at TCP.onread (net.js:556:19)



make: *** [test] Error 5
WDG:ircd.js-master Wes$ 
```

Does anyone know this problem?

Thanks.

--- a/test/user.test.js
+++ b/test/user.test.js
@@ -12,7 +12,7 @@ module.exports = {
         , user = new User(null, server);
 
       assert.ok(!user.hasTimedOut());
-      user.lastPing = (Date.now() - 60001);
+      user.lastPing = (Date.now() - 61000);
       assert.ok(user.hasTimedOut());
 
       done();

